mlrun/projects/pipelines.py

```diff
@@ -742,6 +746,9 @@ class _RemoteRunner(_PipelineRunner):
 
     engine = "remote"
 
+    def __init__(self):
+        self.inner_runner: type[_PipelineRunner] = _KFPRunner
```

Comment:
Where is this being used?
the inner engine is passed as an argument to `get_run_status`, doesn't look like it should be a property

Reply:
right its a leftover from solution. removed